### PR TITLE
Send the plugin version on production environment

### PR DESCRIPTION
**Pull request by gonzunigad**
When the user updates the settings page of the plugin, if the environment is set to PRODUCCION, then the plugins sends the plugin version, the wocoommerce version and the commerce code to a internal telemetry API.

### CODE REVIEW

File: woocommerce-transbank/libwebpay/Telemetry/PluginVersion.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    const ENV_INTEGRATION = 'INTEGRACION';
+    const ENV_PRODUCTION = 'PRODUCCION';
+    const PRODUCT_WEBPAY = 1;
+    
+    const ECOMMERCE_WOOCOMMERCE = 1;
+    const ECOMMERCE_PRESTASHOP = 2;
+    const ECOMMERCE_MAGENTO2 = 3;
+    const ECOMMERCE_VIRTUEMART = 4;
+    const ECOMMERCE_OPENCART = 5;
+    const ECOMMERCE_SDK = 6;
```
Comment (a1b4) [inline woocommerce-transbank/libwebpay/Telemetry/PluginVersion.php]: Why is included the other ecommerce? I think that we should remove the unused constants.

Comment (gonzunigad) [reply woocommerce-transbank/libwebpay/Telemetry/PluginVersion.php]: I want to use the exact same class on other plugins, and eventually export it as a package. This class would be like the 'Internal Telemetry SDK'. At least, that was what i was thinking about when I put all the constants there. Do I remove them anyway?

Comment (gonzunigad) [reply woocommerce-transbank/libwebpay/Telemetry/PluginVersion.php]: cc @a1b4

Comment (a1b4) [reply woocommerce-transbank/libwebpay/Telemetry/PluginVersion.php]: ok I understand, but I need that you create a Jira card for the creation of the package.

**Pull request merged**